src/statements.md

```diff
@@ -28,29 +32,29 @@ const userTypeLiteral = new UserType {
   bar = 3
 }
 const arrayTypeLiteral = new Array<int64> [ 1, 1, 2, 3, 5, 8, 13 ]
-const mapTypeLiteral = new Map<bool, string> {
+const otherArrayTypeLiteral = [ "foo", "bar", "baz" ]
```

Comment:
I wouldn't be sure if this means that `const arrayTypeLiteral = [ 1, 1, 2, 3, 5, 8, 13 ]` is valid

Reply:
Wait, really? Why do you suppose that?

Reply:
I wouldn't suppose that, but I would wonder if `const arrayTypeLiteral = [ 1, 1, 2, 3, 5, 8, 13 ]` is valid or not. Not a big deal now that I come think of it, we can leave as-is

Reply:
:+1: